python/pfs/drp/stella/fitBroadbandSED.py

```diff
@@ -0,0 +1,135 @@
+from lsst.pex.config import Config, Field, DictField
+from lsst.pipe.base import Task
+from lsst.utils import getPackageDir
+
+import numpy
+import numpy.lib.recfunctions
+import astropy.io.fits
+
+import os
+
+
+class FitBroadbandSEDConfig(Config):
+    """Configuration for FitBroadbandSEDTask
+    """
+
+    fluxLibraryPath = Field(
+        dtype=str,
+        default=os.path.join(getPackageDir("drp_pfs_data"), "fluxCalibration",
+                             "syntheticFlux_withInterp_ps_HSCPS1GaiaSDSS.fits"),
+        doc="Synthetic photometry table"
+    )
+
+    filterAliases = DictField(
```

Comment:
I'd call this `filterMappings`: it's a mapping from the pfsConfig filter name to the flux library column. But not really important.